### DISCUSSION TITLE
Support for Yoctopuce Yocto-watt Device

### DISCUSSION THREAD

**Discussion by Hambren** [artisan-roaster-scope/artisan#955] [Ideas]
Good day. Anyone who uses an electric roaster will find the volt/amp/energy monitoring and measurement functions of the Yoctopuce yoctowatt device extremely useful. In particular being able to automate the measurement of cumulative KVAR between bean addition and roast dump points for differing bean moisture contents, size distributions, roast profiles etc etc will, I believe, provide a very valuable future field of experimentation and research - let alone giving improved intuitive guidance as to how to produce the holy grail of a perfect roast all the time, every time! Could you please thus add yoctowatt to the list of available Yoctopuce devices for Artisan as soon as possible? Many thanks.

**Comment by MAKOMO** [comment]
We are open for any code contributions via Pull Requests

**Reply by Hambren** [reply]
Good day Makomo and thanks for your reply. I am clear on what I would like to occur,with this device addition however - and please forgive me - I have absolutely no idea how to go about getting it done - I need to find someone to do it for me. Since your reply I put a job request into Upwork to see if I could find someone to do the change on my behalf, however it has become obvious the changes need to be coded by someone who has working knowledge of Artisan. I am quite happy to pay a reasonable amount to get the device addition done and do believe it will be of broad benefit to the Artisan community. I would really appreciate more detailed guidance from you on how I might be able to connect with someone who knows how to get it done and would be prepared to work with me on it. Kind regards.
